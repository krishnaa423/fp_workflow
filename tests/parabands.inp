input_wfn_file WFN_coo
output_wfn_file WFN_parabands.h5 

vsc_file VSC 
vkb_file VKB 

number_bands 201
