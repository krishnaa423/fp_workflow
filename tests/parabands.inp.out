
 BerkeleyGW version 4.0+git v4.0-1-g1cb27c7 by lzlwell <lzlwell@berkeley.edu> at 2024-06-30 15:14:21 -0500
 ParaBands code, Complex version, run on 17-Jul-2024 at 21:48:18 +0000

                                                                 ..o.          
                                                                .oxxo.         
                                                               .oxxxxo...      
                                                               oxxxxxxo.       
                                                              .oxxxxxxx.       
                                                              .ooooooxxo..     
                                                              .oooooooxo..     
                                                              .oooooxxo...     
                                                       .........oxooo......    
                                                 ............................  
                                             ................................. 
                                          .................................... 
           .          ..oo. ....  .................................oooxxxxxxxo.
     .............oxxxx@ox@@@x@x.....................o...........ooooooooooxx. 
    .o.........oox@x.oo........xxx@@............ooxxxxo..........ooooxxxxxoxo  
    .x........x@xxo...............o@xxo........oxxx@@@xoooooooooooooooxxxo...  
    .o......ox@@o..................oox@o.....ooxxx@xoooxxxxxxxoooooooooooo.... 
    o..ooooo@@xoooo....ooo...........x@o.....ooxxxxo   .oxxxxxxxxxxooooooo.... 
    . .oooo@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@....ooooox.     .oxx@@@@@xxoo........ 
      .ooooxxxxxxxooooooxooooooooooooooxo...oooooxx.        ..ox@xxxoo.........
      .ooooooxxxx@xoooooooooooooxxoooooooooooxxxxx.            .oxxooooooooxxo.
     .oooooxxxxx@@@xxxxxxxxxxxxxxxxxxxxxxxxoxxxxo.                .oxxxxxxxxo. 
   ....oooxxxxx@@@xo..oxxx@@@@@@xxxxoxxoooooooxx.                  .oxxxoo..   
  .....ooxxxx@@xo.       ........    .ooooooooxxo                              
  ..oooxxxx@@@o                       .oooooooxoo.                             
  ....oxooxxxxx.                       .ooo..oooo.                             
  .....o.ooxxxxxo.                     .oooooooxo.                             
 ......ooooxxxxxxo.                     .ooooooxoo..                           
........ooxxxxxxxo..                     .o....oxoo...                         
.......ooooxxxxxxxo.                     ........oooo.                         
.ooooooo..ooxxxxoooo.                    .........ooo...                       
..oxo...ooooxxxoooo..                    .ooo......oooo...                     
  .ooooo....o.                            .oxxxoo....ooo....                   
    .oooooo...                              ...ooooo...ooo..                   
       ...                                     .oo.......                      
                                               ....ooo...                      
                      __              __                                       
 ______              [  |            [  |                 ._____  _          _ 
|_   _               | |  _          | |                /  ___ | |        | |
  | |_) | .---.  _. _.| | / |  .---.  | | .---.  _    _ / /    _|   /  / / 
  |  __'./ /__\[ /`_| '' <  / /__\ | |/ /__\|   | | |   _____  /  / /  
 _| |__| | __. | |   | |`  | ___. | || ___.  / /  .___| |    /  /   
|_______/ .__./[_]  [__|  _] .__./[___].__./    /   .____./    /  /    
                                                  / /                          
                                                 /_/                           
 --------------------------------------------------------------------------------
 Please cite the main BerkeleyGW paper when using results from this calculation:

 Jack Deslippe, Georgy Samsonidze, David A. Strubbe, Manish Jain, Marvin L.
 Cohen, and Steven G. Louie, "BerkeleyGW: A Massively Parallel Computer Package
 for the Calculation of the Quasiparticle and Optical Properties of Materials
 and Nanostructures," Comput. Phys. Commun. 183, 1269 (2012)

 Aditionally, PLEASE CITE ALL PAPERS REPORTED AT THE END OF THIS CALCULATION.
 --------------------------------------------------------------------------------

 Running MPI version (parallel)
 Running with 1 MPI task(s)
 Using OpenMP. Number of threads per MPI task: 16
 Number of threads used for sorting routines: 1

 Compilation flags:
 - Compiler: NVHPC
 - Para. flags: MPI, OMP
 - Math  flags: USESCALAPACK, UNPACKED, USEFFTW3, HDF5
 - Debug flags: 

 Git version: GIT_COMMIT
 Reading parameters from file parabands.inp
 Running with verbosity level 1 (default).

 
 Options for writing output WFN file:
 - Using regular HDF5 write operation
 - Using collective MPI-IO

 Reading self-consistent potential from file VSC
WARNING: Inversion symmetry is present only with a fractional translation.
Apply the translation so inversion is about the origin, to be able to use the real version.
WARNING: Inversion symmetry is present only with a fractional translation.
Apply the translation so inversion is about the origin, to be able to use the real version.
WARNING: Inversion symmetry is present only with a fractional translation.
Apply the translation so inversion is about the origin, to be able to use the real version.
 Reading wavefunctions from file WFN_coo
 Reading k-point-dependent Kleinman-Bylander projectors from file VKB
 Highest occupied band (unshifted grid) = 4
 Valence max    (unshifted grid) = 6.550504 eV
 Conduction min (unshifted grid) = 7.144342 eV
 Middle energy  (unshifted grid) = 6.847423 eV
 Fermi  energy  (unshifted grid) = 6.847423 eV


 Memory available per MPI task: 30325.418 MB

 Problem description:
   FFT grid: 24 24 24
   Size of RHO G-space (gvec%ng): 3287
   Max. number of G-vectors per k-point: 412
   Original number of bands in WFN file: 14
   Number of spins: 1
   Number of spinor components: 1
   Number of k-points: 8
   Max. number of occ states: 4

 Summary of calculation:
   Number of bands to generate: 201
   Number of k-point pools: 1
   Number of MPI ranks per pool: 1
   Number of idle MPI ranks building Ham.: 0
   Number of idle MPI ranks diagonalizing Ham.: 0
   Matrix block size: 64
   Diagonalization algorithm: -1 (ZHEEVX)
 
================================================================================
 21:48:18   Dealing with k =  0.000000  0.000000  0.000000                 1 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 411

 
 Started building local part of H with 26 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 26.
 [ 21:48:18 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:18 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:18 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:18 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 52.
 [ 21:48:18 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:18 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:18 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:18 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:18 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.045 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.403526   -5.490244
    2    0.481453    6.550504
    3    0.481453    6.550504
    4    0.481453    6.550504
    5    0.660769    8.990221
    6    0.660769    8.990221

 Max. difference between input and output mean-field energies: 1.157E-09 Ry
 at (ik, band, spin) = (1, 14, 1).


 Setting up wavefunctions file WFN_parabands.h5 in HDF5 format.

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.050 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.012 s.

================================================================================
 21:48:18   Dealing with k = -0.000000 -0.000000  0.500000                 2 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 410

 
 Started building local part of H with 26 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 26.
 [ 21:48:18 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:18 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:18 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:18 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 52.
 [ 21:48:18 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:18 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:18 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:18 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:18 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.048 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.230646   -3.138104
    2   -0.037429   -0.509241
    3    0.390379    5.311372
    4    0.390379    5.311372
    5    0.590727    8.037249
    6    0.717841    9.766730

 Max. difference between input and output mean-field energies: 2.746E-10 Ry
 at (ik, band, spin) = (2, 4, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.066 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.

================================================================================
 21:48:18   Dealing with k =  0.000000  0.500000 -0.000000                 3 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 410

 
 Started building local part of H with 26 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 26.
 [ 21:48:18 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:18 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:18 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:18 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 52.
 [ 21:48:18 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:18 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:18 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:18 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:18 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.063 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.230646   -3.138104
    2   -0.037429   -0.509241
    3    0.390379    5.311372
    4    0.390379    5.311372
    5    0.590727    8.037249
    6    0.717841    9.766730

 Max. difference between input and output mean-field energies: 2.594E-10 Ry
 at (ik, band, spin) = (3, 4, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.070 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.

================================================================================
 21:48:18   Dealing with k =  0.000000  0.500000  0.500000                 4 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 412

 
 Started building local part of H with 26 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 26.
 [ 21:48:18 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:18 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:18 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:18 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 52.
 [ 21:48:18 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:18 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:18 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:18 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:18 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.047 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.097184   -1.322260
    2   -0.097184   -1.322260
    3    0.265897    3.617717
    4    0.265897    3.617717
    5    0.525099    7.144342
    6    0.525099    7.144342

 Max. difference between input and output mean-field energies: 3.721E-09 Ry
 at (ik, band, spin) = (4, 12, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.070 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.

================================================================================
 21:48:18   Dealing with k =  0.500000  0.000000  0.000000                 5 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 410

 
 Started building local part of H with 26 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 26.
 [ 21:48:18 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:18 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:18 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:18 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 52.
 [ 21:48:18 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:18 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:18 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:18 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:18 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.049 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.230646   -3.138104
    2   -0.037429   -0.509241
    3    0.390379    5.311372
    4    0.390379    5.311372
    5    0.590727    8.037249
    6    0.717841    9.766730

 Max. difference between input and output mean-field energies: 1.062E-10 Ry
 at (ik, band, spin) = (5, 14, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.069 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.

================================================================================
 21:48:18   Dealing with k =  0.500000  0.000000  0.500000                 6 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 412

 
 Started building local part of H with 26 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 26.
 [ 21:48:18 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:18 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:18 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:18 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:18.
 [ 21:48:18 |   0% ] distributed column  1 / 52.
 [ 21:48:18 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:18 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:18 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:18 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:18 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:18 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:18 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:18 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:18 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:18 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:18.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.043 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.097184   -1.322260
    2   -0.097184   -1.322260
    3    0.265897    3.617717
    4    0.265897    3.617717
    5    0.525099    7.144342
    6    0.525099    7.144342

 Max. difference between input and output mean-field energies: 2.495E-09 Ry
 at (ik, band, spin) = (6, 12, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.074 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.

================================================================================
 21:48:19   Dealing with k =  0.500000  0.500000  0.000000                 7 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 412

 
 Started building local part of H with 26 distributed column(s) at 21:48:19.
 [ 21:48:19 |   0% ] distributed column  1 / 26.
 [ 21:48:19 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:19 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:19 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:19 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:19 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:19 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:19 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:19 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:19 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:19.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:19.
 [ 21:48:19 |   0% ] distributed column  1 / 52.
 [ 21:48:19 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:19 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:19 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:19 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:19 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:19 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:19 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:19 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:19 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:19 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:19.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.042 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.097184   -1.322260
    2   -0.097184   -1.322260
    3    0.265897    3.617717
    4    0.265897    3.617717
    5    0.525099    7.144342
    6    0.525099    7.144342

 Max. difference between input and output mean-field energies: 7.631E-10 Ry
 at (ik, band, spin) = (7, 12, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.071 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.

================================================================================
 21:48:19   Dealing with k =  0.500000  0.500000  0.500000                 8 / 8
================================================================================

 Constructing and diagonalizing Hamiltonian H of size 410

 
 Started building local part of H with 26 distributed column(s) at 21:48:19.
 [ 21:48:19 |   0% ] distributed column  1 / 26.
 [ 21:48:19 |   8% ] distributed column  3 / 26, remaining: 0 s.
 [ 21:48:19 |  19% ] distributed column  6 / 26, remaining: 0 s.
 [ 21:48:19 |  27% ] distributed column  8 / 26, remaining: 0 s.
 [ 21:48:19 |  38% ] distributed column 11 / 26, remaining: 0 s.
 [ 21:48:19 |  50% ] distributed column 14 / 26, remaining: 0 s.
 [ 21:48:19 |  58% ] distributed column 16 / 26, remaining: 0 s.
 [ 21:48:19 |  69% ] distributed column 19 / 26, remaining: 0 s.
 [ 21:48:19 |  77% ] distributed column 21 / 26, remaining: 0 s.
 [ 21:48:19 |  88% ] distributed column 24 / 26, remaining: 0 s.
 Finished building local part of H at 21:48:19.
 Elapsed time: 0 s.
 
 
 Started building non-local part of H with 52 distributed column(s) at 21:48:19.
 [ 21:48:19 |   0% ] distributed column  1 / 52.
 [ 21:48:19 |   4% ] distributed column  3 / 52, remaining: 0 s.
 [ 21:48:19 |  10% ] distributed column  6 / 52, remaining: 0 s.
 [ 21:48:19 |  19% ] distributed column 11 / 52, remaining: 0 s.
 [ 21:48:19 |  29% ] distributed column 16 / 52, remaining: 0 s.
 [ 21:48:19 |  38% ] distributed column 21 / 52, remaining: 0 s.
 [ 21:48:19 |  50% ] distributed column 27 / 52, remaining: 0 s.
 [ 21:48:19 |  60% ] distributed column 32 / 52, remaining: 0 s.
 [ 21:48:19 |  69% ] distributed column 37 / 52, remaining: 0 s.
 [ 21:48:19 |  79% ] distributed column 42 / 52, remaining: 0 s.
 [ 21:48:19 |  88% ] distributed column 47 / 52, remaining: 0 s.
 Finished building non-local part of H at 21:48:19.
 Elapsed time: 0 s.
 
 Diagonalizing H.
 Using diagonalization routine: ZHEEVX
 Done diagonalizing H.
 Time elapsed: 0.045 s.

 First 6 eigenvalues:
    #     en (Ry)     en (eV)
    1   -0.230646   -3.138104
    2   -0.037429   -0.509241
    3    0.390379    5.311372
    4    0.390379    5.311372
    5    0.590727    8.037249
    6    0.717841    9.766730

 Max. difference between input and output mean-field energies: 2.610E-10 Ry
 at (ik, band, spin) = (8, 14, 1).

 Writing wavefunctions to file WFN_parabands.h5.
 Done writing wavefunctions.
 Time elapsed: 0.069 s.

 Writing energies to file WFN_parabands.h5.
 Done writing energies and occupations.
 Time elapsed: 0.009 s.


                        CPU (s)        WALL (s)           #

INPUT:                    0.023           0.024           1
OUTPUT:                   0.694           0.692           9
HAMILTONIAN:              0.406           0.405           8

INPUT WFN:                0.015           0.015           1
INPUT VSC:                0.005           0.005           1
INPUT VKB HEAD:           0.003           0.003           1
INPUT VKB DATA:           0.025           0.025           8

OUTPUT HEADER:            0.078           0.078           1
OUTPUT WFN:               0.616           0.614           8

HAM KIN:                  0.000           0.000           8
HAM BUILD LOC:            0.011           0.008           8
HAM BUILD NL:             0.010           0.011           8
HAM BUILD NL COMM:        0.000           0.000          16
HAM DIAG:                 0.382           0.381           8
 
TOTAL:                    1.216           1.216           1
 
